# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/bounds/browser_test_zoom.js
# Commit: 3cfe5bd5d084
# Full Hash: 3cfe5bd5d084baf18f5d95b0e879f6fc179fac34
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-06 04:09:04
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 76a965804210 -r 3cfe5bd5d084 accessible/tests/browser/bounds/browser_test_zoom.js
--- a/accessible/tests/browser/bounds/browser_test_zoom.js	Tue Oct 05 20:24:03 2021 +0000
+++ b/accessible/tests/browser/bounds/browser_test_zoom.js	Tue Oct 05 20:44:46 2021 +0000
@@ -39,7 +39,7 @@
   await testContentBounds(browser, p2);
   await testContentBounds(browser, area);
 
-  await invokeContentTask(browser, [], () => {
+  await SpecialPowers.spawn(browser, [], () => {
     const { Layout } = ChromeUtils.import(
       "chrome://mochitests/content/browser/accessible/tests/browser/Layout.jsm"
     );